cedar-policy-validator/build.rs

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2022-2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I believe Legal signed off on changing this to `Copyright Cedar Contributors` and removing the dates. If we don't want to do that yet, we should use 2024.